src/database.rs

```diff
@@ -230,6 +235,40 @@ impl Database {
                     DbRequest::MarkStuck(request) => {
                         let _unused = request.res.send(mark_stuck(request.order, &orders));
                     }
+                    DbRequest::InitializeServerInfo(res) => {
+                        let server_info_tree = database
+                            .open_tree(SERVER_INFO_TABLE)
+                            .map_err(DbError::DbStartError);
+                        let result = server_info_tree.and_then(|tree| {
+                            if let Some(server_info_data) =
+                                tree.get(SERVER_INFO_ID).map_err(DbError::DbInternalError)?
+                            {
+                                let server_info: ServerInfo =
+                                    serde_json::from_slice(&server_info_data).map_err(|e| {
+                                        DbError::DeserializationError(e.to_string())
+                                    })?;
+                                Ok(server_info.instance_id)
+                            } else {
+                                let mut generator = Generator::default();
+                                let new_instance_id = generator
+                                    .next()
+                                    .unwrap_or_else(|| "unknown-instance".to_string());
```

Comment:
```suggestion
                                    .unwrap();
```
The default generator always returns `Some()`.

Reply:
This could be changed to meaning error in case namegen decides to use some awkward unreliable rng backend in future and we forget it, but this fallback seems reasonable to me at least for now. Or at least expect-style panic would work to explain why we just unwrap here.